.github/workflows/java-cd.yml

```diff
@@ -295,6 +331,9 @@ jobs:
               env:
                   PORT: ${{ steps.port.outputs.PORT }}
               run: |
+                  if [[ "${{ matrix.host.TARGET }}" == *"musl"* ]]; then
+                    export JAVA_HOME=/usr/lib/jvm/java-11-openjdk
+                  fi
```

Comment:
Move this to 'Install git and Java' step, use `$GITHUB_ENV`. Note, java version isn't hard-coded.